wdtk-wikibaseapi/src/main/java/org/wikidata/wdtk/wikibaseapi/WbSearchEntitiesAction.java

```diff
@@ -42,132 +40,130 @@
-     * @param strictLanguage
-     *            (optional) whether to disable language fallback
-     * @param type
-     *            (optional) search for this type of entity
-     *            One of the following values: item, property
-     *            Default: item
-     * @param limit
-     *            (optional) maximal number of results
-     *            no more than 50 (500 for bots) allowed
-     *            Default: 7
-     * @param offset
-     *            (optional) offset where to continue a search
-     *            Default: 0
-     *            this parameter is called "continue" in the API (which is a Java keyword)
-     *
-     * @return list of matching entities retrieved via the API URL
-     * @throws MediaWikiApiErrorException
-     *             if the API returns an error
-     * @throws IllegalArgumentException
-     *             if the given combination of parameters does not make sense
-     */
-    public List<WbSearchEntitiesResult> wbSearchEntities(String search, String language,
-                                                         Boolean strictLanguage, String type, Long limit, Long offset)
-            throws MediaWikiApiErrorException, IOException {
-
-        Map<String, String> parameters = new HashMap<>();
-        parameters.put(ApiConnection.PARAM_ACTION, "wbsearchentities");
-
-        if (search != null) {
-            parameters.put("search", search);
-        } else {
-            throw new IllegalArgumentException(
-                    "Search parameter must be specified for this action.");
-        }
-
-        if (language != null) {
-            parameters.put("language", language);
-        } else {
-            throw new IllegalArgumentException(
-                    "Language parameter must be specified for this action.");
-        }
-        if (strictLanguage != null) {
-            parameters.put("strictlanguage", Boolean.toString(strictLanguage));
-        }
-
-        if (type != null) {
-            parameters.put("type", type);
-        }
-
-        if (limit != null) {
-            parameters.put("limit", Long.toString(limit));
-        }
-
-        if (offset != null) {
-            parameters.put("continue", Long.toString(offset));
-        }
-
-        List<WbSearchEntitiesResult> results = new ArrayList<>();
-
-        JsonNode root = this.connection.sendJsonRequest("POST", parameters);
-        JsonNode entities = root.path("search");
-        for (JsonNode entityNode : entities) {
-            try {
-                JacksonWbSearchEntitiesResult ed = mapper.treeToValue(entityNode,
-                        JacksonWbSearchEntitiesResult.class);
-                results.add(ed);
-            } catch (JsonProcessingException e) {
-                LOGGER.error("Error when reading JSON for entity "
-                        + entityNode.path("id").asText("UNKNOWN") + ": "
-                        + e.toString());
-            }
-        }
-
-        return results;
-    }
```

Comment:
This seems to be a large piece of code that got reformatted in the process, it's not clear to me what changed there?